src/EditorFeatures/CSharpTest/Structure/AnonymousObjectCreationExpressionStructureTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
```

Comment:
💡 These lines typically aren't needed for new files, but it's clear this was just matching the existing form in this component so it's not a problem for the current change.

Reply:
thanks, I didn't know that!